docker-compose.malware-scanner.yml

```
@@ -0,0 +1,50 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+
+  clamav:
+    image: clamav/clamav:stable_base
+    container_name: clamav
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/malware-scanner/clamav/defs:/var/lib/clamav
+      - ./docker/malware-scanner/clamd.conf:/etc/clamav/clamd.conf:ro
```

the name of the folder should reflect the service that is an owner of that directory

```suggestion
      - ./docker/data/clamav/clamav/defs:/var/lib/clamav
      - ./docker/clamav/clamd.conf:/etc/clamav/clamd.conf:ro
```